src/stream/src/common/table/state_table.rs

```diff
@@ -471,9 +471,10 @@ impl<S: StateStore> StateTable<S> {
                     .into_iter()
                     .map(|index| self.pk_indices[index])
                     .collect_vec();
+
                 let read_options = ReadOptions {
-                    prefix_hint: None,
-                    check_bloom_filter: self.dist_key_indices == key_indices,
```

Comment:
Simply modifying `check_bloom_filter` to check whether `dist_key` is a subset of the get `pk` for point lookup is incorrect because in the current implementation, get `pk` will be used to check the bloom filter when `check_bloom_filter` is true.

Reply:
Same issue for the `StorageTable` change.